rfcs/text/0042/risk.yml

```diff
@@ -0,0 +1,144 @@
+---
+- name: risk
+  title: Risk information
+  group: 2
+  short: Fields for describing risk score and level.
+  beta: >
+    These fields are in beta and are subject to change.
+  description: >
+    Fields for describing risk score and risk level of entities such as hosts and users.
+    These fields are not allowed to be nested under `event.*`. Please continue to use
+    `event.risk_score` and `event.risk_score_norm` for event risk.
+  reusable:
+    top_level: false
+    expected:
+      - host
+      - user
```

Comment:
@SourinPaul We're not currently respecting this fact (that `risk` is present under _either_ `host.risk` or `user.risk`) in the persistence implementation, but I'm about to. Before I do I just wanted to verify this behavior: nesting this under one or the other means that we cannot query/aggregate across users _and_ hosts in a single query, they'll pretty much always be two different sets of data that we have to act on independently; it's effectively normalized. As long as we don't need to return both sets of data in a single spot, I think that's fine 👍

Reply:
@rylnd Yes, that's fine. We dont have an explicit search requirement for "Display any entity (host/ user) with risk.level = High".

We have explicit search requirements for each entity type "Display all hosts. all users" in the entity pages. Even if we hit an edge case where we may need to retrieve risk for correlated entities, they can be explicit API calls.